ccpp/data/GFS_typedefs.F90

```diff
@@ -1651,6 +1661,16 @@ module GFS_typedefs
     real (kind=kind_phys), pointer :: tdomip (:)     => null()   !< dominant accumulated sleet type
     real (kind=kind_phys), pointer :: tdoms  (:)     => null()   !< dominant accumulated snow type
 
+    real (kind=kind_phys), pointer :: skebu_wts(:,:) => null()   !< 10 meter u wind speed
```

Comment:
Lines 1664 to 1673 still need to be removed

Reply:
Removed!